server.js

```diff
@@ -2,28 +2,72 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import Thought from './models/thought'
+import thoughtsData from './data/thoughts.json'
+
+const ERR_COULD_NOT_SAVE_THOUGHT = 'Could not save thought to database'
+
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+
+// Created test data to get the database deployed in Mongo Atlas
+if (process.env.RESET_DATABASE) {
+  console.log('Resetting database ...')
+
+  const seedDatabase = async () => {
+    await Thought.deleteMany()
+    await thoughtsData.forEach((thought) => new Thought(thought).save())
+  }
+  seedDatabase()
+}
```

Comment:
It's fine too seed the database if you want to have test data, but you don't have to :)